homeassistant/components/iaqualink/__init__.py

```diff
@@ -149,7 +146,7 @@ async def _async_systems_update(now):
 
             try:
                 await system.update()
-            except AqualinkServiceException as svc_exception:
+            except (AqualinkServiceException, httpx.HTTPError) as svc_exception:
```

Comment:
Side note: If the library raises library specific exceptions it's better to not let other exceptions out but always raise library specific exceptions.

One reason is apparent in this PR. If the library would have had library specific exceptions for the transport errors it could have changed the underlying transport without breaking the integration.

I'd recommend changing the library in this way in the future.